quickstart/pom.xml

```diff
@@ -33,7 +33,7 @@
     </developers>
 
     <properties>
-        <revision>3.1.3</revision> <!-- CHANGE THIS to match project version in the root (not technically parent) pom -->
+        <revision>3.1.4</revision> <!-- CHANGE THIS to match project version in the root (not technically parent) pom -->
```

Comment:
Same as comment above , Quick question : Is it possible to have a root pom version that can be used in all sub projects

Reply:
seems like this kusto-quickstart was intentionally separated not to pick from parent pom. May be @yogilad @asafmah @ohadBitton can comment

Reply:
true 
it is intentionally